sunbird_service_api_test/src/test/resources/templates/user/create/invalidChannelResponse.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
make file name consistence. invalidChannelResponse.json  and user_already_exist_response.json